### PR TITLE
#162163169 Create user profile 

### PR DESCRIPTION
**Pull request by mulondo**
### What does this PR do?
It creates a user profile after the user has registered.

### Description of the task to be completed?
* The user profile is created immediately after the user has registered. 
* When the user profile is created, the bio, image and fun_fact fields are set to be black by default.
* The user can log in and update their user profile.

### How should this be manually tested?
* clone the repository
* type in the terminal "pip install -r requirements.txt" to install the requirements on your local machine.
* git checkout ft-create-userprofile-#162163169.
* make migrations
* run the server and test out the endpoints in postman.

### What are the relevant pivotal tracker stories?
[#162163169](https://www.pivotaltracker.com/story/show/162163169)

### CODE REVIEW

**Review by sekayasin — APPROVED** [review]
looks nice

File: authors/apps/authentication/serializers.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 from rest_framework import serializers
 
 from .models import User
+# from authors.apps.profiles.models import UserProfile
```
Comment (inno-asiimwe) [inline authors/apps/authentication/serializers.py]: Remove commented out code.

**Review by barema4 — APPROVED** [review]
Nice work

**Review by alexkayabula — APPROVED** [review]
Good work.

**Review by ongebo — APPROVED** [review]
Nice work!

**Pull request merged**